wp-content/plugins/tapestry/templates/single-tapestry.php

```diff
@@ -34,7 +34,9 @@
 			<script src="<?php echo plugin_dir_url(__FILE__) ?>tapestry-d3/libs/h5p-resizer.min.js" charset="UTF-8"></script>
 
             <script>
-            	var jsonUrl = "<?php echo plugin_dir_url(__FILE__) ?>tapestry-d3/tapestry.json";
+				var jsonUrl = "<?php echo plugin_dir_url(__FILE__) ?>tapestry-d3/tapestry.json";
```

Comment:
We shouldn't need this anymore, right? Since we are loading the json from the API we are building.

Reply:
This will be removed in another PR that loads tapestry from the database.